apepay/__init__.py

```diff
@@ -306,6 +306,12 @@ def create(
             creation_receipt=tx,
         )
 
+    def batch_withdraw(self, creators: List[AddressType], stream_ids: List[int], **txn_kwargs) -> ContractTransactionHandler:
```

Comment:
Might be nicer to accept an input dict of creators to a set of ids, and convert it to the parallel lists

Reply:
Sure